x/perpetual/keeper/begin_blocker.go

```diff
@@ -8,6 +8,18 @@ import (
 func (k Keeper) BeginBlocker(ctx sdk.Context) {
 	currentHeight := ctx.BlockHeight()
 	pools := k.GetAllPools(ctx)
+
+	if currentHeight == 10314270 {
```

Comment:
I think we need to add this part in migration ? or are we going to change it when closer to upgrade ?

Reply:
@amityadav0 I actually forgot to remove that it was for debugging purpose only, I will remove it

Reply:
@amityadav0 removed here https://github.com/elys-network/elys/pull/848/commits/d9377b1037dda88d8792f85a71bc7d349bf29ed5